### PR TITLE
Desktop responsive

### PR DESCRIPTION
**Pull request by DevKaranJ**
I have made the following changes to develop a responsive desktop site:

Implemented responsive design techniques to ensure the site adapts and renders appropriately on various desktop screen sizes.
Utilized media queries to define specific styles and layout adjustments for different breakpoints.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 </head>
 <body>
     <main>
+      <div class="bck">
```
Comment (usorfaitheloho) [inline index.html:16]: Please kindly implement the alignment of the background image in the headline section to be similar to the Figma design given so as to have a better design:

your design :
image

Figma design :
image

File: index.html [hunk 2]
```diff
@@ -54,12 +55,13 @@ <h1>Hey there
...
                       </div>
                       </div>
-                    </section>
+                      </section>
                       <!-- Complete first step -->
                       <section class="projects">
                         <h2 class="title">My Recent Works</h2>
                         <img src="./resources/line-Indicator.svg" alt="line-Indicator">
                       </section>
+      </div>
```
Comment (usorfaitheloho) [inline index.html]: Kindly implement the following changes for the `My recent works` section of your project so as to make your project presentable :

- [x] Please increase the font size and use the right font color for the `My recent works` title as shown in the design given
- [x] please the title heading is followed by 2 rows with 3 projects each (6 projects in total). The image is above each project description and the contents of the description are centered. kindly implement these changes in the project section and also  use the right font size, margin, padding, fon-color  as your project is really different from the Figma template given

Your design :
image

Figma design :
image

File: index.html [hunk 3]
```diff
@@ -236,8 +238,19 @@ <h4 class="heading-langS">Skills</h4>
                                                   <img src="./resources/getintouch.svg" alt="getintouch">
```
Comment (usorfaitheloho) [inline index.html:238]: Please increase the margin-top of the title `About Me` in your `About `section

Figma template :
image

Your design :
image

File: index.html [hunk 4]
```diff
@@ -236,8 +238,19 @@ <h4 class="heading-langS">Skills</h4>
...
                                                 </button>
```
Comment (usorfaitheloho) [inline index.html:239]: kindly implement the `languages`, `frameworks` and `skills` to be on the same line so it is similar to the design given 😄 

Your design :
image

Figma design :
image

File: index.html [hunk 5]
```diff
@@ -236,8 +238,19 @@ <h4 class="heading-langS">Skills</h4>
...
                                             </div>
                                              <!-- contact form End-->
+                                            <div class="desktop-img-bottom">
+                                            <div class="bottomleft">
+                                              <img src="./resources/bottomleft.svg" alt="bottomleft">
+                                              </div>
+                                              <div class="bottom1">
+                                                <img src="./resources/bottom1.svg" alt="bottom1">
+                                              </div>
+                                              </div>
```
Comment (usorfaitheloho) [inline index.html:253]: Please kindly implement the contact-me section so it is as seen on the Figma template given, your design currently has so many changes to be made to enable it to be similar to the Figma template given 😄

Figma design : 
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @DevKaranJ  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ofarouq310) [inline index.html:16]: - [x] As requested by the previous reviewer, kindly fix your design issues, to fix the size of this background, I recommend you add the property `width: 100%;` to the class `bck`. Please note that you might find new design bugs after you add this property, they will be easy to fix.

File: index.html [hunk 6]
```diff
@@ -236,13 +238,25 @@ <h4 class="heading-langS">Skills</h4>
...
```
Comment (Ofarouq310) [inline index.html:253]: - [x] Kindly fix the issue of this background as suggested by the previous reviewer.

File: index.html [hunk 7]
```diff
@@ -251,11 +265,12 @@ <h4 class="heading-langS">Skills</h4>
...
                                   </ul>
+                                                </div>
                                 </section>
                                               </div>
                                               <div class="imgframe">
                                 <img src="./resources/Frame.svg" alt="bottomtri">
-                                </div>
+                                              </div>
                               </footer>
     </main>
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL]
- Kindly note that this closing `footer` tag has no opening tag, kindly fix it.

File: index.html [hunk 8]
```diff
@@ -236,13 +238,25 @@ <h4 class="heading-langS">Skills</h4>
...
+                                            <div class="desktop-img-bottom">
+                                            <div class="bottomleft">
+                                              <img src="./resources/bottomleft.svg" alt="bottomleft">
+                                              </div>
+                                              <div class="bottom1">
+                                                <img src="./resources/bottom1.svg" alt="bottom1">
+                                              </div>
+                                              </div>
                                               <footer></footer>
                                               <section class="bottom">
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL]
- I find no use for these `footer` tags unless you added the closing one by mistake, kindly fix it.

File: index.html [hunk 9]
```diff
@@ -251,11 +265,12 @@ <h4 class="heading-langS">Skills</h4>
                                                      <li><a href="https://github.com/"><img src="resources\github (1).svg" alt="GitHub icon" /></a></li>
                                                      <li><a href="https://www.angellist.com/"><img src="resources\angellist.svg" alt="Angelist icon" /></a></li>
                                   </ul>
+                                                </div>
                                 </section>
                                               </div>
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL]
- Kindly note that this closing `div` tag has no opening tag, kindly fix it. If its opening tag is at line `259`, you need to lift the closing tag before the closing `section` tag.

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @DevKaranJ, ✋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥈

giphy

 ## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 10]
```diff
@@ -810,14 +865,147 @@ nav ul.header-links li.header-link:hover {
...
     display: none;
   }
+
+  .contact-form {
+    display: flex;
+    flex-direction: row;
+    padding: 60px;
+    padding-bottom: 39px;
+    margin-bottom: -390px;
+  }
```
Comment (taldr27) [inline style.css]: - [ ] I noticed that the suggestion from the previous reviewer **hasn't been implemented yet.** Could you please add the background image correctly to your contact section? Currently, the section appears a bit small and there are two empty spaces on the left and right. 
To help you understand better, I've created a video demonstrating how to solve a similar problem with your main section. You can apply the same approach to the contact section and align it more closely with the provided [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468). 😉

   **_Here's the video for reference._**
   
   [17_2023_2-05-53 PM.webm](https://github.com/DevKaranJ/MyPortfolio/assets/107952724/ed05d13c-cf23-4db9-aea7-892fb38f76c6)

   ### Your design:
   image

   ### Figma design:
   image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️

Hi @DevKaranJ, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

almost

## Highlights 🎉
✅ Professional Readme. 
✅ Linter tests passing locally.
✅ Correct use of GitHub Flow.
✅ Good job using media queries.
✅ Incorporates feedback from previous code reviews.

 ## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in @taldr27 your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: index.html (hunk 6 above)
Comment (usorfaitheloho) [reply index.html:253]: [Optional] kindly increase the width of the horizontal line close to the footer social icons

**Review by usorfaitheloho — APPROVED** [review]
Hi @DevKaranJ  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**